src/components/TransactionModalContent.jsx

```diff
@@ -1,18 +1,49 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { Button } from 'antd';
 import Lottie from 'lottie-react';
 
 import TnxFailedAnimation from '../assets/tnx-failed.json';
 import TnxSubmittedAnimation from '../assets/tnx-submitted.json';
 import TnxSuccessfulAnimation from '../assets/tnx-successful.json';
+import config from '../configuration';
+import CommunityService from '../services/CommunityService';
+import ErrorHandler from '../lib/ErrorHandler';
+
+const DeepPurpleColor = { color: '#2C0B3F' };
+const contentTextStyle = {
+  color: '#6B7087',
+  fontSize: '18px',
+  fontFamily: 'Lato',
+};
 
 const TransactionModalContent = ({ type, txUrl, isCommunity, msg }) => {
   let title = '';
-  let description = '';
+  let description;
   let animation;
   let loop = false;
 
+  const TnxSuccessfulDescription = () => (
+    <Fragment>
+      Your transaction has been processed.
+      <a href={txUrl} target="_blank" rel="noopener noreferrer">
+        {' '}
+        View on Etherescan
+      </a>
+    </Fragment>
+  );
+
+  const gotoDefaultCommunity = () => {
+    const { defaultCommunityId } = config;
+    CommunityService.getByDelegateId(defaultCommunityId)
```

Comment:
`defaultCommunityId` can be empty or zero, in this case donate to default community (Giveth here) should not be available